itests/testdata/issue_smart_asset_tx_testdata.go

```diff
@@ -0,0 +1,266 @@
+package testdata
+
+import (
+	f "github.com/wavesplatform/gowaves/itests/fixtures"
+	utl "github.com/wavesplatform/gowaves/itests/utilities"
+)
+
+const (
+	IssueSmartAssetMinVersion = 2
+)
+
+func GetPositiveAssetScriptData(suite *f.BaseSuite) map[string]IssueTestData[ExpectedValuesPositive] {
+	return map[string]IssueTestData[ExpectedValuesPositive]{
+		"Valid script, true as expression": *NewIssueTestData(
```

Comment:
Ubiquitous de-referencing :)
What you think about returning type, not pointer to type from `NewIssueTestData`?

Reply:
Will be changed in new PR. Need to review all such places.